docs/DeveloperGuide.md

```diff
@@ -325,6 +325,46 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use Case: Edit an internship entry.**
+
+**MSS**
+
+1.  User edits an internship entry.
+2.  InternBuddy updates that particular internship entry.
+
+    Use case ends
+
+**Extensions**
+
+* 1a. InternBuddy detects that index of internship entry given is not a number.
+  * 1a1. InternBuddy prompts user for a valid internship entry index.
+  
+    Use case resumes from Step (1).
+
+* 1b. InternBuddy detects that index of internship given is out of range.
```

Comment:
I feel that it is a bit vague to say out of range. Is it better to write "an internship entry is not found using the given user-supplied argument"?